dts/bindings/sensor/ti,ina23x-common.yaml

```
@@ -34,10 +34,10 @@ properties:
       while keeping a good measurement resolution. The units are in uA/LSB
       so that low maximum currents can be measured with enough resolution.
 
-  rshunt-milliohms:
+  rshunt-microohms:
```

```suggestion
  rshunt-micro-ohms:
```

per dtschema convention

---

Thanks @gmarull, Changes made.